_posts/patterns/2016-01-28-alerts.md

```diff
@@ -0,0 +1,108 @@
+---
+layout:         pattern
+title:          Alerts
+date:           2015-01-28 00:00:00
+
+categories:     patterns
+tags:
+- atomic
+- pattern
+- alerts
+
+slug:           alerts
+
+url_styles:     "patterns/_alerts.scss"
+
+description:    Examples of alerts in a semi-real world context.
+
+info:           Alerts provide information about...TBD
+---
+
+<h3 class="hd-6 example-set-hd">Error Alert with Title and Button</h3>
+<div class="example-set">
+    <div class="alert alert-error" role="alert" aria-labelledby="alert-title-error" tabindex="-1">
+        <span class="icon alert-icon icon-exclamation-triangle" aria-hidden="true"></span>
+
+        <div class="alert-message">
+            <h3 class="alert-title" id="alert-title-error">
+                We are unable to load this chart.
+            </h3>
+        </div>
+
+        <button type="button" class="alert-action btn-neutral btn">Read More</button>
```

Comment:
@clrux Might this be an appropriate place to use '.sr' text?  I could make a case for not wanting to visually add more text than needed to the button.